scripts/delayed.js

```diff
@@ -733,7 +733,22 @@ function injectFevoScript() {
 
   return fevoWidget;
 }
+
+function injectWeFevoScript() {
+  loadScript('https://offer.fevo.com/js/fevo.js', () => { }, {
+    type: 'text/javascript',
+  });
+}
+
+// there are 2 distinct types of fevo buttons
+// and we need to support both
+// see decorateFevoButtons in scripts.js
 const hasFevo = document.querySelector('a.fevo-btn');
 if (hasFevo) {
   injectFevoScript();
 }
+
+const hasWeFevo = document.querySelector('a.we-fevo-btn');
+if (hasWeFevo) {
```

Comment:
`if (hasFevo || hasWeFevo)`?

Reply:
I don't think so?

Not at all confident this is right, but my understanding from Gary is that there are 2 types of fevo buttons, but they aren't related beyond the fact that they come from the same vendor. but also that there could at least in theory exist both types of buttons on the same page. 

so they presence of a "normal" fevo button does not necessitate loading the sdk for the "we fevo" type, and vice versa.

Reply:
Wow. I definitely read this wrong. Thank you.